app/src/main/java/uk/co/ribot/androidboilerplate/util/MyGsonTypeAdapterFactory.java

```diff
@@ -0,0 +1,12 @@
+package uk.co.ribot.androidboilerplate.util;
+
+
+import com.google.gson.TypeAdapterFactory;
+import com.ryanharter.auto.value.gson.GsonTypeAdapterFactory;
+
+@GsonTypeAdapterFactory
+public abstract class MyGsonTypeAdapterFactory implements TypeAdapterFactory{
```

Comment:
This will not pass our internal checkstyle tests as we require a space before the opening of braces {
`implements TypeAdapterFactory {`